frontend/sw360-portlet/src/main/resources/META-INF/resources/html/utils/includes/searchPackages.jsp

```diff
@@ -0,0 +1,214 @@
+                                <span class="alert alert-info my-0 p-2 mx-4"><liferay-ui:message key="only.orphan.packages.can.be.searched.and.linked.to.a.release" />.</span>
+                            </core_rt:if>
+                            <span id="pkgTruncationAlert" class="alert alert-warning my-0 p-2 mx-4" role="alert" style="display: none;">
+                                <liferay-ui:message key="output.limited.to.100.results.please.narrow.your.search" />
+                            </span>
+                        </div>
+                        <div id="search-package-form">
+                            <div class="spinner text-center" style="display: none;">
+                                <div class="spinner-border" role="status">
+                                    <span class="sr-only"><liferay-ui:message key="loading" /></span>
+                                </div>
+                            </div>
+                            <table id="packageSearchResultsTable" class="table table-bordered">
+                                <colgroup>
+                                    <col style="width: 3%;" />
+                                    <col style="width: 30%;" />
+                                    <col style="width: 10%;" />
+                                    <col style="width: 15%;" />
+                                    <col style="width: 12%;" />
+                                    <col style="width: 30%;" />
+                                </colgroup>
+                                <thead>
+                                    <tr>
+                                        <th></th>
+                                        <th><liferay-ui:message key="name" /></th>
+                                        <th><liferay-ui:message key="version" /></th>
+                                        <th><liferay-ui:message key="license" /></th>
+                                        <th><liferay-ui:message key="package.manager" /></th>
+                                        <th><liferay-ui:message key="purl" /></th>
+                                    </tr>
+                                </thead>
+                                <tbody>
+                                </tbody>
+                            </table>
+                        </div>
+                    </form>
+                </div>
+                <div class="modal-footer">
+                <button type="button" class="btn btn-light" data-dismiss="modal"><liferay-ui:message key="close" /></button>
+                    <button id="selectPackageButton" type="button" class="btn btn-primary" title="<liferay-ui:message key="link.packages" />"><liferay-ui:message key="link.packages" /></button>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<script>
+require(['jquery', 'modules/dialog', 'bridges/datatables', 'utils/keyboard', /* jquery-plugins */ 'jquery-ui' ], function($, dialog, datatables, keyboard) {
```

Comment:
May be `/* jquery-plugins */ 'jquery-ui'` can be removed, seems like they are not used.